attributes/default.rb

```diff
@@ -52,3 +54,10 @@
 default['onlinefs']['kafka_consumer']['topic_pattern']    = ".*_onlinefs"
 default['onlinefs']['kafka_consumer']['topic_list']       = ""
 default['onlinefs']['kafka_consumer']['poll_timeout_ms']  = 1000
+default['onlinefs']['kafka_consumer']['ron_db_group_id']     = "0"
+default['onlinefs']['kafka_consumer']['vector_db_group_id']  = "1"
```

Comment:
while not a big deal it could help with understanding if group ids were informative. We could call them `0` -> `onlinefs_rondb` and `1` -> `onlinefs_vectordb`